apps/backend/src/models/lib/item.model.js

```
@@ -112,15 +116,22 @@ ItemSchema.pre("save", function (next) {
+        this.completedAt = new Date();
+    }
+
     next();
 });
 
 ItemSchema.pre("findOneAndUpdate", function (next) {
     const update = this.getUpdate();
     if (update.$set && update.$set.status === "done") {
         update.$set.isCompleted = true;
+        update.$set.completedAt = new Date();
     } else if (update.$set && update.$set.status) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)